babel.cfg

```diff
@@ -0,0 +1 @@
+[python: navigator_engine/**.py]
```

Comment:
Do we still need this file now that we are not translating any code strings in engine?

Reply:
Babel is only used for its `get_locale` function. I've removed the config file